osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -585,6 +579,32 @@ private void newItemsPresented(IEnumerable<CarouselItem> carouselItems)
                 carousel.NextRandom();
         }
 
+        private IDisposable? noResultsDuck;
```

Comment:
This is going to fail. Can you guess why?

Reply:
This is addressed by detaching/re-attaching ducking on screen transition events.